### PR TITLE
support for hipchat

### PR DESCRIPTION
**Pull request by rajatjindal**
Support for hipchat. issue #56 

I want to make more changes to how to start the service, will open another PR with those changes.

Kindly review and let me know if any comments.

### CODE REVIEW

**Comment by sebgoa** [comment]
@ngtuna can you do a quick review

File: kubewatch-in-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kubewatch
+  namespace: default
+spec:
+  containers:
+  - image: tuna/kubewatch:v0.0.1
```
Comment (sebgoa) [inline kubewatch-in-cluster.yaml]: we need to build a new image fwiw

File: pkg/handlers/hipchat/hipchat.go [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+*/
+
+package hipchat
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	hipchat "github.com/tbruyelle/hipchat-go/hipchat"
```
Comment (sebgoa) [inline pkg/handlers/hipchat/hipchat.go]: you need to vendor this dependency

**Comment by rajatjindal83** [comment]
Hi @sebgoa , thanks for the quick review.

I am wondering if you have any opinion about migrating this project to use 'dep' instead of 'godep' for dependencies.

Thanks
Rajat Jindal

**Pull request merged**